### DISCUSSION TITLE
Add classification in the dashboard to show  minor and major updates separated.

### DISCUSSION THREAD

**Discussion by marinated-wings** [renovatebot/renovate#9054] [Request Help]
Hello All,

I'm trying the dashboard for the first time and i have some questions:
Is it possible to have a separation between minor and major updates:
example: 

Minor:
* [ ] Update dependency @angular/animations to v11.2.4
* [ ] Update dependency @angular/cdk to v11.2.3
-----------------------------------------------
Major:
* [ ] Update dependency @angular/common to v11.2.4
* [ ] Update dependency @angular/compiler to v11.2.4

or at least to add a better details in the naming to include label or any kind of indication if it is minor or major .

Thank you

**Comment by rarkins** [comment]
Major updates won't look like in your example, because they will read like "to v4" not "to v4.1.2". They are also sorted last after minor updates.
